src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -123,27 +123,35 @@
                                    (repeatedly children #(random-structured-fact (rand-nth (range depth))
                                                                                  (rand-nth (range children)))))})))
 
+(defn update-catalog [catalog stamp]
```

Comment:
Seems like this function should have a different name or maybe get slurped into where it's used. Right now it's a pretty generic fn name for something pretty specific.